js/bybit.js

```diff
@@ -2987,9 +3316,6 @@ module.exports = class bybit extends Exchange {
         }
         let type = undefined;
         [ type, params ] = this.handleMarketTypeAndParams ('fetchOrder', market, params);
```

Comment:
@sc0Vu aren't we missing the `unifiedMargin`check inside `fetchOrder` ?

Reply:
How do you think call `fetchOrders` directly?

Reply:
@sc0Vu Yeah I think we can do that by providing the `orderId`